regolith/helpers/l_grantshelper.py

```diff
@@ -64,30 +64,19 @@ def sout(self):
         rc = self.rc
         grants = []
         desired_date = None
-        both = False
-        if not rc.date and not rc.current:
-            both = True
         if rc.date:
-            desired_date = get_datetime(rc.date)
+            desired_date = date_parser.parse(rc.date).date()
+        elif rc.current:
```

Comment:
I think it is cleaner if you just make this ``else:``

Then ``desired_date`` is always defined and you don't need the if logic down below